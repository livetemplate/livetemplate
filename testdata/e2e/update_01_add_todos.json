{"s":["<div class=\"container\">\n        <header>\n            <h1>","</h1>\n            <div class=\"counter\">Count: ","</div>\n            <div class=\"status ","\">\n                Status: ","\n            </div>\n        </header>\n        \n        <main>\n            <section class=\"stats\">\n                <h2>Statistics</h2>\n                <p>Total Todos: ","</p>\n                <p>Completed: ","</p>\n                <p>Remaining: ","</p>\n                <p>Completion Rate: ","</p>\n            </section>\n            \n            <section class=\"todos\">\n                <h2>Todo List</h2>\n                ","\n            </section>\n        </main>\n        \n        <footer>\n            <p>Last updated: ","</p>\n            <p>Session ID: ","</p>\n        </footer>\n    </div>"],"0":"Task Manager","1":"3","2":"inactive","3":"Low Activity","4":"3","5":"1","6":"2","7":"33%","8":{"d":[{"0":"","1":"k1pskw","2":"#0","3":"Learn Go templates","4":"○","5":"(Priority: High)"},{"0":"completed","1":"k1142q","2":"#1","3":"Build live updates","4":"✓","5":"(Priority: Medium)"},{"0":"","1":"k1aqr4","2":"#2","3":"Write documentation","4":"○","5":"(Priority: Low)"}],"s":["\n                    <div class=\"todo-item ","\" data-lvt-key=\"","\">",">\n                        <strong>",":</strong> \n                        "," \n                        ","\n                        ","\n                    </div>\n                    "]},"9":"2023-01-01 10:15:00","10":"session-12345"}