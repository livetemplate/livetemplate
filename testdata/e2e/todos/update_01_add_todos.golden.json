{
  "0": "Task Manager",
  "1": "3",
  "10": "session-12345",
  "2": "inactive",
  "3": "Low Activity",
  "4": "3",
  "5": "1",
  "6": "2",
  "7": "33%",
  "8": "\u003cdiv class=\"todo-list\"\u003e\n                    \n                    \u003cdiv class=\"todo-item \" data-key=\"todo-1\"\u003e\n                        \u003cstrong\u003e#0:\u003c/strong\u003e\n                        Learn Go templates\n                        ○\n                         (Priority: High)\n                    \u003c/div\u003e\n                    \n                    \u003cdiv class=\"todo-item completed\" data-key=\"todo-2\"\u003e\n                        \u003cstrong\u003e#1:\u003c/strong\u003e\n                        Build live updates\n                        ✓\n                         (Priority: Medium)\n                    \u003c/div\u003e\n                    \n                    \u003cdiv class=\"todo-item \" data-key=\"todo-3\"\u003e\n                        \u003cstrong\u003e#2:\u003c/strong\u003e\n                        Write documentation\n                        ○\n                         (Priority: Low)\n                    \u003c/div\u003e\n                    \n                \u003c/div\u003e",
  "9": "2023-01-01 10:15:00",
  "s": [
    "\u003cdiv class=\"container\"\u003e\u003cheader\u003e\u003ch1\u003e",
    "\u003c/h1\u003e\u003cdiv class=\"counter\"\u003eCount: ",
    "\u003c/div\u003e\u003cdiv class=\"status ",
    "\"\u003e Status: ",
    " \u003c/div\u003e\u003c/header\u003e\u003cmain\u003e\u003csection class=\"stats\"\u003e\u003ch2\u003eStatistics\u003c/h2\u003e\u003cp\u003eTotal Todos: ",
    "\u003c/p\u003e\u003cp\u003eCompleted: ",
    "\u003c/p\u003e\u003cp\u003eRemaining: ",
    "\u003c/p\u003e\u003cp\u003eCompletion Rate: ",
    "\u003c/p\u003e\u003c/section\u003e\u003csection class=\"todos\"\u003e\u003ch2\u003eTodo List\u003c/h2\u003e ",
    " \u003c/section\u003e\u003c/main\u003e\u003cfooter\u003e\u003cp\u003eLast updated: ",
    "\u003c/p\u003e\u003cp\u003eSession ID: ",
    "\u003c/p\u003e\u003c/footer\u003e\u003c/div\u003e"
  ]
}